lib/modules/manager/terragrunt/modules.ts

```diff
@@ -87,7 +87,10 @@ export function analyseTerragruntModule(
       }
     } else {
       // The packageName should only contain the path to the repository
-      dep.packageName = pathname.replace(/^\//, '').split('//')[0];
+      dep.packageName = pathname
+        .replace(/^\//, '')
```

Comment:
```suggestion
        .replace(regEx(/^\//), '')
```
use our regex helper